### PR TITLE
create list payment component

### PR DESCRIPTION
**Pull request by DragonMeme**
## Pull Request Information
**Relates to Github Issue:** #40, #42

**What is the PR Type:** 
Feature

**Describe the added/fixed behaviour:**
Payments page can now render the required data from backend (both payments associated with the user and contributions).
~~Although work to be done is to be able to obtain the JWT auth token without hard-coding the token on the Authorization header.~~
EDIT: Authorization is now supported.

## Checklist

- [x] Pull request has suitable title, adequately described and associated with a github issue
- [x] Branch name follows correct convention (e.g. feature/[id]-some-feature)
- [x] Branch builds and executes with no errors
- [x] Test cases has been generated for the feature and test cases pass
- [ ] Updated relevant wiki/documentation

## Final Reminders
Please remember to squash the commits

### CODE REVIEW

**Comment by DragonMeme** [comment]
> 
> 
> Please get rid of hard coded auth token, i will help you with this later

Done!

File: Frontend/frontend/src/Components/Payments.js [hunk 1]
```diff
@@ -54,6 +26,70 @@ export default class Payments extends Component {
...
+                            let user_list = [];
+                            for(let i = 0; i < UsersAPIResult.length; i++) 
+                                user_list.push(UsersAPIResult[i]["userName"]);
+                            this.addUserPayment(user_list);
+                        }
+                    )
+                }
+            )
+        })
+    }
```
Comment (ZianYangAng) [inline Frontend/frontend/src/Components/Payments.js]: This should be moved to the APIRequest class in the util folder. 
On line 35 you should probably use the GetAuthString method from the APIRequest class as well. 
If I were to offer a suggestion when this is moved to that APIRequest class I would have a variable called authString which would be put in the header request.
E.g.
`const authString = await APIRequest.getAuthString();`
Then on line 35 and 57, I would change the code to 
`Authorization: authString`

Comment (Richardng12) [reply Frontend/frontend/src/Components/Payments.js]: Good explanation !

Comment (DragonMeme) [reply Frontend/frontend/src/Components/Payments.js]: API Requests now moved from `Payments.js` to `APIRequests.js` as requested.

**Comment by DragonMeme** [comment]
> 
> 
> Most things LGTM, but one other slight change would be variable names.
> 
> So I think the convention for variables in this project is camel case. As an example, in the Payments component, the state `Payments` would be `payments` and `ContributionPayments` would also change to `contributionPayments`.
> 
> One last slight change would be on line 43 in the payments component where there is a variable `list_user` which would change to `listUser` with camel case.
> 
> Everything else though is looking good. 👍

Done (maybe almost) 👍

**Comment by DragonMeme** [comment]
Wierd, though .db files are git ignored

**Comment by Richardng12** [comment]
> 
> 
> Wierd, though .db files are git ignored

Try delete testdb.db before merging

**Comment by DragonMeme** [comment]
Done!

**Pull request merged**